Validate integration parameters display name (#1074)
demisto/demisto-sdk: CHANGELOG.md

* Added support for the incident fields in outgoing mappers in the ID set.
* Added a validation that the taskid field and the id field under the task field are both from uuid format and contain the same value.
* Updated the **format** command to generate uuid value for the taskid field and for the id under the task field in case they hold an invalid values.

> * Added validation that integration parameter display names are capitalized and spaced using whitespace and not underscores.

Reviewer: ```suggestion
* Added validation that integration parameter's display names are capitalized and spaced using whitespaces and not underscores.
```

Author: done

> * Added a validation that the integration parameters display name is capitalized and spaced using whitespace and not underscores.

Reviewer 2: ```suggestion
* Added validation for integration parameter display names are capitalized and spaced using whitespace and not underscores.
```

Author: done
